src/errorMsgs.js

```diff
@@ -125,7 +161,37 @@ const findErrors = (function () {
     return checkSyntax;
 }());
 
-
+const tableDirectives = [
+       'api'
+      ,'audit','auditcols',//'audit cols','audit columns'
+      ,'check'
+      ,'colprefix'
+      ,'compress','compressed'
+      ,'insert'
+      ,'rest'
+      ,'select'
+      ,'unique' , 'uk'
+      ,'pk'
+      ,'cascade','setnull' //'set null'
+];
+
+const columnDirectives = [
+       'idx','index','indexed'
+      ,'unique','uk'
+      ,'check'
+      ,'constant'
+      ,'default'
+      ,'values'
+      ,'upper'
+      ,'lower'
+      ,'nn','not'//,'not null'
+      ,'between'
+      ,'hidden','invisible'
```

Comment:
Same comment about you removing this from docs but still here

Reply:
fixed